pip/pep425tags.py

```diff
@@ -198,7 +247,8 @@ def get_supported(versions=None, noarch=False):
                 supported.append(('%s%s' % (impl, versions[0]), abi, arch))
 
         # Has binaries, does not use the Python API:
-        supported.append(('py%s' % (versions[0][0]), 'none', arch))
+        for arch in arches:
+            supported.append(('py%s' % (versions[0][0]), 'none', arch))
```

Comment:
@dstufft this change also impact macosx: previously it would use the `arch` from `get_platform()` instead of all possible `arches` expanded in this function. I think this is a bug fix but I am not 100% sure. I could not find a test for this.